addons/l10n_es_modelo130/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+def _add_mod130_tax_tags(env):
+    taxes_repartition_lines = env['account.tax.repartition.line'].search([
+        ('tax_id.name', 'like', 'WH'),
+        ('tax_id.country_code', '=', 'ES'),
+        ('tax_id.type_tax_use', '=', 'sale'),
+    ])
+    invoice_mod130_tax_tag = env['account.account.tag'].search([('name', '=', '-mod130[06]')])
+    refund_mod130_tax_tag = env['account.account.tag'].search([('name', '=', '+mod130[06]')])
+
+    for line in taxes_repartition_lines:
+        if line.repartition_type == 'tax':
+            if line.document_type == 'invoice':
+                line.write({'tag_ids': invoice_mod130_tax_tag})
```

Comment:
no need to use write for one elem, we can directly do line['tag_ids'] = ... i think